src/routes.js

```diff
@@ -77,6 +78,11 @@ const Stack = createStackNavigator();
           component={ScreenAbout}
         />
       </Stack.Navigator>
+      <Stack.Screen
```

Comment:
`Stack.Screen` tem que está dentro do `</Stack.Navigator>`, não?

Pelo que eu entendi essa tela vai ser usada em um Tab na parte de baixo da tela, desta forma vc pode retirar a stackscreen das rotas por enquanto não esta usando. Mas se for deixar, favor colocar dentro do stackNAvigator.

Reply:
Corrigido. Se possível me confirme o recebimento das novas alterações.